docs/HowToBuild.html

```diff
@@ -8,7 +8,12 @@ <h1 class="pageTitle">Building FormIt JavaScript Plugins</h1>
 <h2 class="pageSubtitle">Learn How to Build, Test, and Host Your Own Plugins</h2>
 <p class="pageBlurb">FormIt's JavaScript plugin infrastructure allows the creation of custom 3D workflows and bespoke modeling functionality, using a variety of UI types and a powerful API.</p>
 
-<h3 class="categoryDetailTitle">Jump to...</h3>
+<h3 class="categoryDetailTitle">Jump to (First time plugin developers)...</h3>
```

Comment:
This is a good idea, to have a separate jump to section for the two potential types of devs. Nice!